crates/chain/src/spk_iter.rs

```diff
@@ -250,6 +249,14 @@ mod test {
             SpkIterator::new_with_range(&no_wildcard_descriptor, 1..=2).next(),
             None
         );
+        assert_eq!(
```

Comment:
I'm not sure why these tests are being added, but I won't complain about adding new tests 😛